scripts/image-builder/build.sh

```diff
@@ -153,16 +179,30 @@ if [ ${build_disk} -lt 10485760 ] ; then
 fi
 
 title "Downloading local OpenShift and MicroShift repositories"
-# Copy microshift local RPM packages
+# Copy MicroShift RPM packages
 rm -rf microshift-local 2>/dev/null || true
-cp -TR "${ROOTDIR}/../../packaging/rpm/_rpmbuild/RPMS" microshift-local
+if [[ ${MICROSHIFT_RPM_SOURCE} == http* ]] ; then
```

Comment:
```suggestion
if [[ "${MICROSHIFT_RPM_SOURCE}" == http* ]] ; then
```

To avoid issues with (unlikely) space paths